llvm/lib/Target/Mips/MipsAsmPrinter.cpp

```diff
@@ -72,6 +72,11 @@ using namespace llvm;
 
 extern cl::opt<bool> EmitJalrReloc;
 
+cl::opt<bool>
```

Comment:
What purpose does this option serve that is not already covered by the Driver and SubtargetFeature?

Reply:
A way to disable it if need arises. Mostly for the sake of LTO where feature gets baked in object files, so to disable the feature you need to make sure that all object files are build w/o feature enable whereas you can just disable it in the backend regardless of how the object files get built.

I would like to drop this though in favor of your assembler changes if those changes go with emitting relocations for li[48] and addiu[gp][48] when immediate may pose the trouble. We might even do that unconditionally and make the driver option be a pass trough for the linker.